### PR TITLE
fix gradient

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by johngruen** [comment]
Self merging since this is changing a hex value

File: frontend/src/styles/_theme.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $theme-dark: $blue-90;
-$theme-light: rgba(37, 0, 62, .8);
```
Comment (johngruen) [inline frontend/src/styles/_theme.scss:2]: @youwenliang changing this to the same value w/o an opacity layer since that made the gradient look stripe-y

Comment (youwenliang) [reply frontend/src/styles/_theme.scss:2]: sure! sounds good

**Comment by SoftVision-PaulOiegas** [comment]
@johngruen @fzzzy It seems that this was merged on dev right after the stage server deploy. Can we cherry-pick this somehow so it reaches production also if we deploy today?

**Comment by SoftVision-PaulOiegas** [comment]
Also, the stripes are still visible in some of the areas but it's clearly an improvement.

**Pull request merged**